onchain/ProvethVerifier.sol

```diff
@@ -190,26 +186,19 @@ contract ProvethVerifier {
         bytes32 blockHash,
         bytes proofBlob
     ) internal returns (uint8 result, uint256 index, Transaction memory t) {
-        result = TX_PROOF_RESULT_INVALID;
```

Comment:
Not sure why you're getting rid of this? I generally favor descriptive names to returning an integer, but it's probably fine either way

Reply:
I got rid of `TX_PROOF_RESULT_INVALID`, since we now `revert` on invalid proofs.
But I could probably kick out 
```
result = 0;
index = 0;
``` 
since that's the default anyways.